.github/workflows/build.yml

```diff
@@ -6,92 +6,35 @@ on:
   pull_request:
 
 jobs:
-  build-master:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        idf_target: ["esp32", "esp32s2", "esp32s3"]
-    steps:
-    - name: Checkout repo
-      uses: actions/checkout@v2
-      with:
-        submodules: 'recursive'
-    - name: esp-idf build
-      uses: espressif/esp-idf-ci-action@main
-      with:
-        target: ${{ matrix.idf_target }}
-        path: 'examples'
-
-  build-release-v5_0:
-    name: Build for ${{ matrix.idf_target }} on ${{ matrix.idf_ver }}
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        idf_ver: ["release-v5.0"]
-        idf_target: ["esp32", "esp32s2", "esp32s3"]
-    steps:
-    - name: Checkout repo
-      uses: actions/checkout@v2
-      with:
-        submodules: 'recursive'
-    - name: esp-idf build
-      uses: espressif/esp-idf-ci-action@main
-      with:
-        esp_idf_version: ${{ matrix.idf_ver }}
-        target: ${{ matrix.idf_target }}
-        path: 'examples'
-
-  build-release-v4_4:
+  build-examples:
     name: Build for ${{ matrix.idf_target }} on ${{ matrix.idf_ver }}
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        idf_ver: ["v4.4"]
+        idf_ver: ["release-v4.1", "release-v4.2", "release-v4.3", "release-v4.4", "release-v5.0", "release-v5.1", "master"]
```

Comment:
"master" image can not be found. I think we generally do not need it.

Reply:
Oh, it must be `latest`. Do you want me to remove it or to change it to latest?

Reply:
Good question... let's try with it and I can set it as not critical for CI to pass